backend/dataall/base/utils/enum_utils.py

```diff
@@ -0,0 +1,26 @@
+from dataall.base.api import gql
+
+
+def generate_enum_resolver(enum_class):
+    def resolver(context, source):
+        return [{'name': item.name, 'value': item.value} for item in enum_class]
+
+    return resolver
+
+
+EnumItem = gql.ObjectType(
+    name='EnumItem',
+    fields=[
+        gql.Field(name='name', type=gql.String),
+        gql.Field(name='value', type=gql.String),
+    ],
+)
+
+
+def generate_enum_query(enum_class, test_scope):
+    return gql.QueryField(
+        name=enum_class.__name__,
```

Comment:
* Should we get `args` as input?
* Do you think it makes sense to allow requesting multiple enums in one request to minimize RTT?

Reply:
Args? What for?
No, I don't think so. Usually, it's at most two enums per page with no heavy resolver. May be we can save it for later.

Reply:
Ah wait my bad, I think I misunderstood your implementation. What you do is generate a query for each enum that's why you don't require an input argument.

Would it maybe be simpler to create a query with an input argument `enum_names` that points to a resolver that does the lookup for the enums in `GraphQLEnumMapper.__subclasses__()`? If you don't want to expose everything you can add an allow-list in the config.json but I don't see why.

Reply:
With this dynamic lookup we can get a circular dependencies issue. Every module depends on" base". In order to do that, we have to create another module, I'm not sure it worth it.

Reply:
It worked)